### PR TITLE
WIP: Test from cfgov-refresh

### PR DESCRIPTION
**Pull request by hillaryj**
Now supports automatic testing from cfgov-refresh using `python
manage.py test regcore` to test satellite apps.

The tests for this repo have broken, due to what looks like a change to
haystack. Error is `ImportError: cannot import name connection_router`
due to `from haystack import connection_router, connections`.

We’re moving to regs 2.0 in the near future and the tests all pass from
cfgov-refresh, so I’m making a PR even though some tests don’t run
properly.

### CODE REVIEW

File: regcore/settings/base.py [hunk 1]
```diff
@@ -54,3 +55,6 @@
     from local_settings import *
 except ImportError:
     pass
+
+import django
+django.setup()
```
Comment (hillaryj) [inline regcore/settings/base.py]: I've removed those lines and they are not necessary after all. The settings overrides for tests took care of the issue this was fixing - I was trying to get the original regcore tests running in their repo after the fixes required for cfgov satellite apps.

I'll get with you and find out which test errors you're getting. Thanks for taking a look at this!

**Comment by hillaryj** [comment]
@chosak when you run `python manage.py test` inside the `regulations-core` repo, do you still get errors from `regcore_write` and `regcore_read`?

**Comment by chosak** [comment]
@hillaryj yes, these tests have been able to pass using `python manage.py test` from `master` without error, even before this PR. They continue to pass with this PR.

**Comment by hillaryj** [comment]
Okay, sounds like this is also not necessary for now.

**Pull request closed without merging**